chain/client/src/stateless_validation/partial_witness/partial_witness_tracker.rs

```diff
@@ -134,24 +129,7 @@ impl CacheEntry {
             )),
         };
 
-        match decode_result {
```

Comment:
this is moved to the top level because we want to mark witness as "processes" when decoding fails since such part could only be produced by a malicious chunk producer